### ISSUE TITLE
Test on my own data with ground-truth masks

### ISSUE THREAD

**Issue by isadoramsouza** [DrSleep/tensorflow-deeplab-resnet#101]
Hi! After change the number of classes (i used 2, background-black and foreground-white) and training the network with the Pascal dataset, how can I test it with my own test data with ground-truth masks (binary masks)....how is it possible? I want to calculate for exemple the accuracy, precision, recall of the segmentation..sorry about the dumb question, I am new here :(

**Comment by DrSleep** [comment]
refer to readme
